modules/renter/repair.go

```diff
@@ -201,6 +208,14 @@ func (r *Renter) managedRepairIteration(rs *repairState) {
 		}
 	}
 
+	// Accept work non-blocking if there is any
```

Comment:
This code is not non-blocking, but I don't think we want it to be non-blocking either. The code looks good just the comment is incorrect.